Subject: [PATCH] 10_0_fix
---
Index: src/main/webapp/WEB-INF/jsp/users.jsp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/webapp/WEB-INF/jsp/users.jsp b/src/main/webapp/WEB-INF/jsp/users.jsp
--- a/src/main/webapp/WEB-INF/jsp/users.jsp	(revision 94c1392466da99cf39dd8549a0901e6e10ab629d)
+++ b/src/main/webapp/WEB-INF/jsp/users.jsp	(date 1681155554321)
@@ -79,7 +79,7 @@
 <jsp:include page="fragments/footer.jsp"/>
 </body>
 <script type="text/javascript">
-    const i18n = [];
+    const i18n ={}; // https://learn.javascript.ru/object
     i18n["addTitle"] = '<spring:message code="user.add"/>';
     i18n["editTitle"] = '<spring:message code="user.edit"/>';
 
